godot-macros/src/util/mod.rs

```diff
@@ -243,8 +245,18 @@ pub(crate) fn extract_cfg_attrs(
     attrs: &[venial::Attribute],
 ) -> impl IntoIterator<Item = &venial::Attribute> {
     attrs.iter().filter(|attr| {
-        attr.get_single_path_segment()
-            .is_some_and(|name| name == "cfg")
+        let Some(attr_name) = attr.get_single_path_segment() else {
+            return false;
+        };
+        // #[cfg(condition)]
+        if attr_name == "cfg" {
+            return true;
+        }
+        // #[cfg_attr(condition, attributes...)], note that there can be multiple attributes seperated by comma.
+        if attr_name == "cfg_attr" && attr.value.to_token_stream().to_string().contains("cfg(") {
+            return true;
+        }
+        false
```

Comment:
Please add empty lines before comments that document  multi-line statements:
```suggestion
        let Some(attr_name) = attr.get_single_path_segment() else {
            return false;
        };
        
        // #[cfg(condition)]
        if attr_name == "cfg" {
            return true;
        }
        
        // #[cfg_attr(condition, attributes...)], note that there can be multiple attributes seperated by comma.
        if attr_name == "cfg_attr" && attr.value.to_token_stream().to_string().contains("cfg(") {
            return true;
        }
        
        false
```